src/test/java/teammates/test/cases/browsertests/StudentProfilePageUiTest.java

```diff
@@ -332,4 +333,12 @@ private StudentProfilePage getProfilePageForStudent(String studentId) {
         return loginAdminToPage(profileUrl, StudentProfilePage.class);
     }
 
+    /**
+     * Test data removed due to {@link BackDoor#removeAndRestoreDataBundle(DataBundle)}}.
+     */
+    @AfterClass
+    public void classTearDown() {
+        BackDoor.removeDataBundle(testData);
```

Comment:
We need to explain a little more that because one test can run before another we need to remove both of them.

Also, it might be simpler that we instead place this test in sequential tests. However, it doesn't seem like listeners is working properly in IntelliJ. Does it work for you?